Modify Datacenter to Datacenter code list

andrasfabian12 opened the pull request:
Change log:
  - Add GNSS network as a codeList
  - Modify Datacenter to DataCenterCodeType
  - Add an example for the datacenter codeListType

umeat commented on schemas/geodesyml/0.6/monumentInfo.xsd:
```diff
@@ -179,7 +179,8 @@ Copyright: Scripps Orbit and Permanent Array Center (SOPAC), Commonwealth Govern
                 <sequence>
-                    <element maxOccurs="unbounded" name="dataCenter" type="string"/>
+                    <element maxOccurs="unbounded" name="dataCenter" type="geo:DataCenterCodeType"/>
+	                <element minOccurs="0" maxOccurs="unbounded" name="gnssNetwork" type="geo:NetworkCodeType"/>
```
This appears to be indented one too many times. Should be at same level as dataCenter.

umeat commented on schemas/geodesyml/0.6/monumentInfo.xsd:
```diff
@@ -179,7 +179,8 @@ Copyright: Scripps Orbit and Permanent Array Center (SOPAC), Commonwealth Govern
-                    <element maxOccurs="unbounded" name="dataCenter" type="string"/>
+                    <element maxOccurs="unbounded" name="dataCenter" type="geo:DataCenterCodeType"/>
+	                <element minOccurs="0" maxOccurs="unbounded" name="gnssNetwork" type="geo:NetworkCodeType"/>
                     <element name="urlForMoreInformation" type="string"/>
```
If dataCenter is a URL, I'm not sure this is needed.

umeat replied:
I suppose it isn't necessarily? In some examples, it is a URL, and in others it is not.

andrasfabian12 commented:
Thank you for your comments. I tried to improve a bit.
Changes:
- Updated Network/Datacenter type to gml:CodeType.
- Removed the codelist.xml file.
- Updated the examples accordingly.
- Removed unnecessary tabulators.

@umeat : Please review it and let me know if you still have an issue in the code.
